ibm/service/power/data_source_ibm_pi_instances.go

```diff
@@ -267,3 +277,20 @@ func flattenPvmInstanceNetworks(list []*models.PVMInstanceNetwork) (networks []m
 	}
 	return
 }
+
+func flattenPvmInstanceFault(fault *models.PVMInstanceFault) map[string]interface{} {
+	faultMap := make(map[string]interface{})
+	if fault.Code != 0 {
+		faultMap[Attr_Code] = strconv.FormatFloat(fault.Code, 'f', -1, 64)
+	}
```

Comment:
Setting it directly to `Code` makes sense as the attribute value should be "0" instead of ""/null.

Reply:
Updated